### PR TITLE
Adding solution for second exercise

### PR DESCRIPTION
**Pull request by anshu-kumarr**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (gagan93jtg) [inline .vscode/settings.json]: why do we need this file?

File: assign2/index.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="col-1">
+        <div class="head">
+          <h1>Why Loconav's Tracking System?</h1>
+        </div>
+        <div class="d1">
+          <img id="i1" src="topl.png" />
```
Comment (gagan93jtg) [inline assign2/index.html]: is this id `i1` telling me that I'm referring to the first image?

File: assign2/index.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+            <input type="text" placeholder="Phone Number" /><br />
+            <input type="text" placeholder="How Many Vehicles?" />
+            <div class="sel">Select Your Vehicle</div>
+            <div class="options">
+              <button>Car</button>
+              <button>Truck</button>
+              <button>Others</button>
+            </div>
+            <button class="submit-btn">GET STARTED</button>
+            <p>Call Us: +91 8448881414</p>
```
Comment (gagan93jtg) [inline assign2/index.html]: Check font of this text from the design and match it. It's not Raleway

File: assign2/index.html [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+            <p></p>
```
Comment (gagan93jtg) [inline assign2/index.html]: Why this blank tag? Just for spacing? Can't we use something else here?

File: assign2/style.css [hunk 5]
```diff
@@ -0,0 +1,133 @@
+body {
+	margin: 0;
```
Comment (gagan93jtg) [inline assign2/style.css]: why 4 space indentation here?

File: assign2/style.css [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+	margin-left: 170px;
+	margin-bottom: 50px;
+}
+
+.col-1 {
+	width: 60%;
+	margin-left: 50px;
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: space-between;
```
Comment (gagan93jtg) [inline assign2/style.css]: why did we use this property? What happens if we don't use this?

File: assign2/style.css [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+	width: 40%;
+}
+
+.form {
+	padding: 10px 10px;
+	text-align: center;
+	width: 400px;
+	height: 470px;
+	margin: 100px auto;
+	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
```
Comment (gagan93jtg) [inline assign2/style.css]: As I can see from the design, it's equal box-shadow on all four sides. But in your case, it is not the same.

File: assign2/style.css [hunk 8]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+input {
+	width: 64%;
+	padding: 10px;
+	margin: 10px 0px;
+}
+
+.sel {
+	margin: 0;
```
Comment (gagan93jtg) [inline assign2/style.css]: can we write this in a single line? Properties like margin and padding can be mentioned in a single line for all four sides

File: assign2/style.css [hunk 9]
```diff
@@ -0,0 +1,133 @@
...
+	background-color: rgb(42, 172, 177);
+}
+
+img {
+	width: 150px;
+	height: 100px;
+}
+
+a {
+	text-decoration: none;
```
Comment (gagan93jtg) [inline assign2/style.css]: again a different spacing scheme (tab based). Pls use 2 space consistently

File: assign2/style.css [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+
+input {
+	border: 1px solid rgb(225, 225, 225);
+}
+
+button:hover {
+	cursor: pointer;
+}
+
+.d1 {
```
Comment (gagan93jtg) [inline assign2/style.css]: can we merge this like?

```css
.d1, .d2, .d3, .d4 {
  width: 50%;
}
```

File: assign2/style.css [hunk 11]
```diff
@@ -0,0 +1,133 @@
...
+
+.d3 {
+	width: 50%;
+}
+
+.d4 {
+	width: 50%;
+}
+
+#i1 {
```
Comment (gagan93jtg) [inline assign2/style.css]: similarly here (improve class name though)

File: style.css [hunk 12]
```diff
@@ -14,17 +14,23 @@ body{
 }
  input{
    width: 175px;
-   margin: 10px 5px;
-   border : none;
-   border-bottom: 0.4px solid black;
+   padding: 5px;
+   margin: 20px 5px;
+   border-color: rgb(213, 207, 207);
+   border-style: none none solid none;
```
Comment (gagan93jtg) [inline style.css:20]: 👍

File: style.css [hunk 13]
```diff
@@ -14,17 +14,23 @@ body{
...
   margin-top: 20px;
   padding: 5px;
-  width : 175px;
+  width : 200px;
   border-radius: 20px;
+}
+.pi,.ui{
+  padding-bottom: 5px;
+  vertical-align: middle;
 }
```
Comment (gagan93jtg) [inline style.css]: check how to remove this red symbol

**Review by gagan93jtg — CHANGES_REQUESTED** [review]

File: assign2/index.html (hunk 3 above)
Comment (anshu-kumarr) [reply assign2/index.html]: sir it is not changing even if I am specifically selecting this p tag and adding font-style.
so I changed to his style as its is similar
.contact {
  font-family: sans-serif;
  font-weight: 400;
}

File: assign2/index.html (hunk 4 above)
Comment (anshu-kumarr) [reply assign2/index.html]: removed

File: assign2/index.html (hunk 2 above)
Comment (anshu-kumarr) [reply assign2/index.html]: changed to img-1

File: .vscode/settings.json (hunk 1 above)
Comment (anshu-kumarr) [reply .vscode/settings.json]: we don't need this file. I have deleted this file

File: assign2/style.css (hunk 9 above)
Comment (anshu-kumarr) [reply assign2/style.css]: ok sir done

File: assign2/style.css (hunk 10 above)
Comment (anshu-kumarr) [reply assign2/style.css]: all four div had same  width that's why I used compound selector

File: assign2/style.css (hunk 6 above)
Comment (anshu-kumarr) [reply assign2/style.css]: sir row wrap to make all div align with row and if any div does not fit in that row it should automatically move to next row and justify-content to provide space between the div.

**Review by gagan93jtg — APPROVED** [review]

**Pull request merged**